spec/features/admin/overview_spec.rb

```diff
@@ -3,31 +3,18 @@
 feature %q{
   As a backend user
   I want to be given information about the state of my enterprises, products and order cycles
-} , js: true do
+}, js: true do
   include AuthenticationWorkflow
   include AuthorizationHelpers
   include WebHelper
 
-  stub_authorization!
-
   context "as an enterprise user" do
-    before :each do
+    before do
       @enterprise_user = create_enterprise_user
       Spree::Admin::OverviewController.any_instance.stub(:spree_current_user).and_return @enterprise_user
       quick_login_as @enterprise_user
     end
 
-    context "with no enterprises" do
```

Comment:
Is this outdated? Just to test if I got it right:

- People create the first enterprise during the signup process.
- If they haven't done that, the admin panel redirects to /unauthorized.

Correct?